### PR TITLE
Remove nick editor

### PR DESCRIPTION
**Pull request by xPaw**
Depends on #2229

### CODE REVIEW

**Comment by astorije** [comment]
Lets remove the nick entirely here, like shown [here](https://user-images.githubusercontent.com/113730/38126670-0acb853a-33c0-11e8-94f7-cc2657acc59c.png) or [your Solarized theme](https://github.com/thelounge/thelounge-theme-solarized/blob/master/screenshot.png). Removing it was part of my design update and I still think we should do that.

**Comment by xPaw** [comment]
I didn't remove it entirely because it's probably beneficial in some cases (e.g. public mode). We could hide it by default, and make it visible in public or those that want to show it with custom css.

**Comment by astorije** [comment]
Sure, let's do that.

File: client/js/utils.js [hunk 1]
```diff
@@ -83,17 +82,8 @@ function join(args) {
...
-	$("#nick-value").prop("contenteditable", toggle);
-}
-
 function setNick(nick) {
-	// Closes the nick editor when canceling, changing channel, or when a nick
-	// is set in a different tab / browser / device.
-	toggleNickEditor(false);
-
-	$("#nick-value").text(nick);
+	$("#nick").text(nick);
```
Comment (astorije) [inline client/js/utils.js]: I believe this 1-line function is now called in 2 places left, and I don't think we'll have any further case. Move its content to where it's called instead?

**Comment by xPaw** [comment]
Yes.

**Pull request merged**